components/details/MetadataTags.tsx

```diff
@@ -0,0 +1,67 @@
+import { memo } from "react";
+import Head from "next/head";
+import { useRouter } from "next/router";
+
+export interface MetadataShareTagsProps {
+  title: string;
+  description: string;
+  shareTitle?: string;
+  shareDescription?: string;
+  shareImage?: string;
+  type?: string;
+}
+
+const MetadataTags = ({
+  title,
+  description,
+  shareTitle,
+  shareDescription,
+  shareImage,
+  type,
+}: MetadataShareTagsProps) => {
+  if (shareTitle === undefined) {
+    shareTitle = title;
+  }
+
+  if (shareDescription === undefined) {
+    shareDescription = description;
+  }
+
+  if (shareImage === undefined) {
+    shareImage = `${process.env.NEXT_PUBLIC_HOST}/logo-sharecard.jpg`;
+  }
+
+  if (type === undefined) {
+    type = "website";
+  }
+
+  const pageUrl = `${process.env.NEXT_PUBLIC_HOST}${useRouter().asPath}`;
+
+  // Next needs parts of all the page Head in one place for the meta tags to appear properly.
+  // PageUrl route needs to come from outside this function.
+  return (
+    <Head>
+      {/* Page Details */}
+      <title>{title}</title>
+      <meta name="description" content={description} />
+
+      {/* Metadata for Twitter sharing */}
+      <meta name="twitter:card" content="summary_large_image" />
+      <meta name="twitter:site" content="@COKIproject" />
```

Comment:
We should also add this tag: `<meta name="twitter:creator" content="@COKIproject" />`